README.md

```diff
@@ -14,7 +14,7 @@ While there is not yet a stable release, the current version in beta testing is:
 <dependency>
   <groupId>com.metamx</groupId>
   <artifactId>rad-tech-datatypes</artifactId>
-  <version>0.0.3</version>
+  <version>0.0.5</version>
```

Comment:
hmm I may need to update that because there may have been some other changes since this branch was created